src/walletSdk/Anchor/index.ts

```diff
@@ -19,21 +43,23 @@ export class Anchor {
   }
 
   async getInfo(): Promise<TomlInfo> {
-    const toml = await StellarTomlResolver.resolve(this.homeDomain);
-    return parseToml(toml);
+    const stellarToml = await StellarTomlResolver.resolve(this.homeDomain);
+    const parsedToml = parseToml(stellarToml);
+    this.toml = parsedToml;
+    return parsedToml;
   }
 
   async auth() {
-    const tomlInfo = await this.getInfo();
-    return new Auth(tomlInfo.webAuthEndpoint);
+    const toml = this.toml || await this.getInfo();
```

Comment:
[nit]
Maybe move this logic to getInfo?

Reply:
@Ifropc agree on that

I was actually on the fence on moving this logic to getInfo to make sure getInfo would always fetch fresh TOML values.
But as Anchors should rarely update their TOML having it cached probably won't be an issue in favor of wallet performance. At any rate I think we could have the best of both worlds by adding a boolean input to the getInfo function so that devs could force refresh the TOML whenever needed. Something like this:
```
  async getInfo(shouldRefresh?: boolean): Promise<TomlInfo> {
    if(this.toml && !shouldRefresh) {
      return this.toml;
    }

    const stellarToml = await StellarTomlResolver.resolve(this.homeDomain);
    const parsedToml = parseToml(stellarToml);
    this.toml = parsedToml;
    return parsedToml;
  }
```

What do you think?

Reply:
Just updated PR with above change